fstab.yaml

```diff
@@ -1,2 +1,2 @@
 mountpoints:
-  /: https://adobe.sharepoint.com/sites/FedsPublishing/Shared%20Documents/fedPub
+  /: https://adobe.sharepoint.com/sites/testnarcis/Shared%20Documents/documents
```

Comment:
wee should keep the original sharepoint site; `testnarcis` was only used for the draft

Reply:
Sure, just made the change